osac-installer/charts/osac-devstack/files/install-virt.sh

```
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+# dev-full: install the virtualization stack on Kind — Multus CNI, KubeVirt, CDI.
+# Requires KUBECONFIG to point at the kind cluster (set by the Makefile).
+#
+# Usage: install-virt.sh <kind-cluster-name>
+
+set -euo pipefail
+
+SCRIPT_DIR="$(CDPATH= cd -- "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)